scripts/trento-support.sh

```diff
@@ -33,50 +34,50 @@ collect_base_system() {
     helm version
 
     echo "#==[ Command ]======================================#"
-    echo "# $(which helm) get hooks $INSTALLATION_NAME"
-    helm get hooks $INSTALLATION_NAME
+    echo "# $(which helm) get hooks $RELEASE_NAME"
+    helm get hooks $RELEASE_NAME
```

Comment:
`helm` commands as well have the `-n` `--namespace` option. I would include it.
It is true, that our installation script doesn't have that option, so I guess it will install in the `default` namespace.
@abravosuse would it make sense to add this flag to the installation script as well?